### PR TITLE
[ADD][10.0]hr_attendance_reason

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
Odoo removed this feature from v9 onwards. IMO this feature is still useful. Employees may have some issues that make them not to register their attendance. With this feature the can notify the HR manager. The manager will set the reason of the manual attendance.

### CODE REVIEW

File: hr_attendance_reason/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    'category': 'Human Resources',
+    'website': 'https://github.com/OCA/hr',
+    'author': 'Odoo S.A.,'
+              'Odoo Community Association (OCA)',
+    'license': 'LGPL-3',
+    'installable': True,
+    'depends': [
+        'hr_attendance',
+    ],
+    'data': [
```
Comment (nikul-serpentcs) [inline hr_attendance_reason/__manifest__.py:18]: @aheficent Data sequence should be [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L29)

**Review by nikul-serpentcs — COMMENTED** [review]
Minor Change

File: hr_attendance_reason/__manifest__.py (hunk 1 above)
Comment (AaronHForgeFlow) [reply hr_attendance_reason/__manifest__.py:18]: Thanks! @nikul-serpentcs

**Review by bjeficent — APPROVED** [review]
LGTM :+1:

File: hr_attendance_reason/tests/test_hr_attendance_reason.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        self.att_reason = self.att_reason_model.create(
+            {'name': 'Bus did not come',
+             'code': 'BB',
+             'action_type': 'sign_in'})
+
+    def test_employee_edit(self):
+        att = self.att_model.create(
+            {'action': 'sign_in',
+             'employee_id': self.employee.id,
+             'name': time.strftime("%Y-%m-01 %H:%M:%S")})
```
Comment (nikul-serpentcs) [inline hr_attendance_reason/tests/test_hr_attendance_reason.py]: @aheficent Here use `default server Data time`

**Review by nikul-serpentcs — COMMENTED** [review]
Minor Change

File: hr_attendance_reason/tests/test_hr_attendance_reason.py (hunk 2 above)
Comment (AaronHForgeFlow) [reply hr_attendance_reason/tests/test_hr_attendance_reason.py]: Done. Thanks.

**Comment by AaronHForgeFlow** [comment]
@nikul-serpentcs Can you please update your review? Thanks

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM :+1:

**Comment by AaronHForgeFlow** [comment]
Rebased and squashed. All good.

**Pull request merged**